src/containers/FeedInfoContainer.ts

```diff
@@ -27,13 +28,15 @@ const mapStateToProps = (state: AppState, ownProps: { navigation: TypedNavigatio
     };
 };
 
-const updateNavParam = (feeds: Feed[], navigation: TypedNavigation) => {
+const getFeedToOpen = (feeds: Feed[], navigation: TypedNavigation) => {
     const feedUrl = navigation.getParam<'FeedInfo', 'feed'>('feed').feedUrl;
     const isFollowed = navigation.getParam<'FeedInfo', 'feed'>('feed').followed;
 
     const updatedFeed = feeds.find(feed => feed.feedUrl === feedUrl);
     if (updatedFeed != null && updatedFeed.followed !== isFollowed) {
```

Comment:
Why don't you put the `updatedFeed.followed !== isFollowed` condition in the `find`? To me it's easier to understand that way.

Reply:
resolved in 272386a